parsing_minmax.py

```diff
@@ -0,0 +1,61 @@
+##Developing a Database that tells us the min&max repeat sizes in samples for each locus
+#Columns: repeat	min_size	max_size	min_size_sample	max_size_sample (5 columns)
+
+##Steps
+#Iterate over each line and group everything by chromosome loci
+#for each allele (2 of them per sample), print out the length
+#For each loci, extract the min and max allele lengths, along with their sampleIDs (5 values, loci, min, max, sampleIDmin, sampleIDmax)
+#Create a dataframe of this information (pandas) and sort them based on chromosome location (ascending order)
+
+
+import gzip
+import itertools
+import numpy as np
+import pandas as pd
+from collections import namedtuple
+
+RepeatRec = namedtuple("RepeatRec", "sample short_allele long_allele")
+
+#Parse through file and extract the repeat information (sample, alleles' size) along with their chrom location
```

Comment:
Can you replace these comments with docstrings? E.g.
`"""organize information to retrieve min and max length alleles for each loci, along with their sample IDs"""`
Would go under the function definition. See [example](https://github.com/ccmbioinfo/crg2/blob/master/scripts/get_HPO_pedigree.py#L8).